File: PYTHON/Program/Ayah.py
from Orang import Orang

class Ayah(Orang):
    def __init__(self, Nama ="", Usia = 0, Jenis_kelamin = "", Pekerjaan = "", Pendidikan =""):
        super().__init__(Nama, Usia, Jenis_kelamin)
        self.Pekerjaan = Pekerjaan
        self.Pendidikan = Pendidikan
      

    # getter dan setter
    def getPekerjaan(self):
        return self.Pekerjaan

    def setPekerjaan(self, Pekerjaan):
        self.Pekerjaan = Pekerjaan

    def getPendidikan(self):
        return self.Pendidikan

    def setPendidikan(self, Pendidikan):
        self.Pendidikan = Pendidikan
    
    #untuk menampilkan data ayah
    def print_ayah(self):
        print(f"Nama          : {self.nama}")
        print(f"Usia          : {self.usia} tahun")
        print(f"Jenis Kelamin : {self.jenis_kelamin}")
        print(f"Pekerjaan     : {self.Pekerjaan}")
        print(f"Pendidikan    : {self.Pendidikan}")

File: CPP/Program/Ayah.cpp
#pragma once
#include <iostream>
#include <string>
#include <vector>
#include "Orang.cpp"
using namespace std;

class Ayah : public Orang
{
private:
    string Pekerjaan;
    string Pendidikan;

public:
    //construct
    Ayah()
    {
    }
    Ayah(string nama, int usia, string jenis_kelamin, string pekerjaan, string pendidikan) : Orang(nama, usia, jenis_kelamin)
    {
        this->Pekerjaan = pekerjaan;
        this->Pendidikan = pendidikan;
    }

    //getter dan setter
    string getPekerjaan()
    {
        return Pekerjaan;
    }

    void setPekerjaan(string Pekerjaan)
    {
        this->Pekerjaan = Pekerjaan;
    }

    string getPendidikan()
    {
        return Pendidikan;
    }

    void setPendidikan(string Pendidikan)
    {
        this->Pendidikan = Pendidikan;
    }

    //untuk menampilkan data ayah
    void printAyah()
    {

        cout << "Nama          : " << getNama() << endl;
        cout << "Usia          : " << getUsia() << " tahun" << endl;
        cout << "Jenis Kelamin : " << getJenis() << endl;
        cout << "Pekerjaan     : " << getPekerjaan() << endl;
        cout << "Pendidikan    : " << getPendidikan() << endl;
    }
};